demo/app/main-page.xml

```diff
@@ -29,6 +29,7 @@
      <TabViewItem title="Buttons">
        <TabViewItem.view>
           <StackLayout className="mjk-buttons">
+          <Button text="logout" tap="{{  }}">
```

Comment:
@govi2010 There seems to be an issue here. The Button was created but never closed.

Reply:
I know @jogboms. I was trying to implement logout button for google.

Reply:
Lets me make this work and complete this PR. Sorry for Delay but It too much work on my side.